transcendence-app/src/user/dto/update-user.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { IsEmail, IsOptional } from 'class-validator';
+
+export class UpdateUserDto {
+  @IsEmail()
+  @IsOptional()
+  email?: string;
```

Comment:
We could add more options to this DTO when we add more fields to the user table.

Reply:
Why is `UpdateUserDto` better than this `Partial<User>`? They seem largely overlapping in function and content

Reply:
`Partial<User>` allows the users to update all the fields from the `Users` table. Including `avatarId`, `createdAt`, and any column we might add in the future.
This class limits the fields the users can update.